File: 2st/Riyeon/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int n = arr.length>arr[0].length? arr.length: arr[0].length;
        int[][] answer = new int[n][n];
        for(int i=0; i<arr.length; i++){
            for(int j=0; j<arr[i].length; j++)
                answer[i][j] = arr[i][j];
        }
        return answer;
    }
}

File: 2st/Riyeon/정사각형으로 만들기.py
def solution(arr):
    answer = []
    n = max(len(arr), len(arr[0]))
    
    for i in range(n):
        li = []
        for j in range(n):
            if j < len(arr[0]) and i < len(arr):
                li.append(arr[i][j])
            else:
                li.append(0)
        answer.append(li)
    return answer